server/src/server.js

```diff
@@ -1,337 +1,189 @@
-let http = require("http"),
-  path = require('path'),
-  url = require('url'),
-  fs = require('fs'),
-  React = require("react"),
-  Router = require("react-router"),
-  routes = require("./routes.js"),
-  models = require("./models.js"),
-  lookupContentType = require('mime-types').contentType,
-  git = require('git-rev'),
-  Cookies = require("cookies"),
-  qs = require('querystring');
+const path = require('path');
+const Cookies = require("cookies");
 
 const { Shot } = require("./servershot");
 const { checkLogin, registerLogin } = require("./users");
 const dbschema = require("./dbschema");
+const express = require("express");
+const bodyParser = require('body-parser');
+const morgan = require("morgan");
+const linker = require("./linker");
```

Comment:
I don't like the comma form, it's harder to edit and rearrange, each line doesn't stand on its own.